common/json_helpers.c

```diff
@@ -281,10 +281,12 @@ void json_add_address(struct json_stream *response, const char *fieldname,
 		json_add_string(response, "type", "torv3");
 		json_add_string(response, "address", fmt_wireaddr_without_port(tmpctx, addr));
 		json_add_num(response, "port", addr->port);
+#if EXPERIMENTAL_FEATURES /* BOLT7 DNS RFC #911 */
 	} else if (addr->type == ADDR_TYPE_DNS) {
```

Comment:
(GH won't let me comment on a *commit*, so I'm commenting on these lines).

I would remove the Changelog-Added from the prior commit, and use Changelog-EXPERIMENTAL: here instead.